go/vt/sqlparser/token.go

```diff
@@ -334,8 +334,16 @@ func (tkn *Tokenizer) Scan() (int, []byte) {
 		switch ch {
 		case eofChar:
 			return 0, nil
-		case '=', ',', ';', '(', ')', '+', '*', '%', '&', '|', '^', '~':
+		case '=', ',', ';', '(', ')', '+', '*', '%', '&', '^', '~':
 			return int(ch), nil
+		case '|':
+			switch tkn.lastChar {
```

Comment:
Ok. I saw instances for cases like <= and implemented in a similar fashion for ||. Will update as per @sougou recommendation